src/contracts/Qearn.h

```diff
@@ -234,13 +253,19 @@ struct QEARN : public ContractBase
 
     PUBLIC_PROCEDURE_WITH_LOCALS(Lock)
     
-        if(qpi.invocationReward() < MINIMUM_LOCKING_AMOUNT) return ;
+        if(qpi.invocationReward() < MINIMUM_LOCKING_AMOUNT) 
+        {
+            output.returnCode = -1;
```

Comment:
you need to do this for other `return` in this function and `Unlock` as well.
Please add a comments somewhere about returnCode meanings

Reply:
added the comments